core/src/main/java/com/netflix/conductor/core/orchestration/ExecutionDAOFacade.java

```diff
@@ -323,7 +329,7 @@ int pruningIntervalInMinutes() {
      * Stale workflows are ocnsidered not normal and are left due to unforeseen circumstances. Stale
      * entries looged need to be investigated.
      */
-    public void pruneWorkflowsAndTasks() {
+    private void pruneWorkflowsAndTasks() {
```

Comment:
changing this to private, hope this will not break anything, for example if this method was invoked on-demand.

Reply:
i verified that its not called from other pleaces.